src/http/api/views/account.posts.view.ts

```diff
@@ -433,7 +433,11 @@ export class AccountPostsView {
 
                 if (activity.object.tag && activity.object.type === 'Note') {
                     const mentionedAccounts: Mention[] = [];
-                    for await (const tag of activity.object.tag) {
+                    const tags = Array.isArray(activity.object.tag)
+                        ? activity.object.tag
+                        : [activity.object.tag];
+
+                    for await (const tag of tags) {
```

Comment:
The `await` here shouldn't be necessary

Reply:
Idea: we could filter mentions tag before entering the loop to increase readability, e.g.:

```
  const mentions = tags.filter(tag => tag.type === 'Mention');

  for (const mention of mentions) {
      const account = await this.getMentionedAccount(
          new URL(mention.href),
          mention.name
      );

      if (!isError(account)) {
          mentionedAccounts.push(getValue(account));
      }
  }
  ```